manifests/config.pp

```diff
@@ -245,14 +245,7 @@
     notify => Class['keycloak::service'],
   }
 
-  file { "${keycloak::install_base}/${_subdir}/configuration":
```

Comment:
You'll want to keep this resource and remove `$_server_conf_dir` from `$_dirs` because this resource has more restricted permissions since it will contain configs.

Reply:
Also you can just call this resource `$_server_conf_dir`.